### PR TITLE
Cli test more

### PR DESCRIPTION
**Pull request by dheles**
Thank you for contributing to KBSecret! Please fill out the items below to help us merge
your work more quickly.

- [X] Have you run `make test` locally and ensured that all tests pass?
- [X] Have you run `make coverage` locally and ensured that coverage did not drop?

*If* you're changing something in the library:
- [ ] Have you introduced unit tests for your changes?

*If* you're changing something in the CLI:
- [ ] Have you updated the manual pages and shell completion (if necessary)?
- [ ] Have you introduced CLI unit tests for your changes and ensured that `make test-cli` passes?

Please describe your changes briefly here. Thank you!

several more cli tests

### CODE REVIEW

File: test/cli/test_helper.rb [hunk 1]
```diff
@@ -3,3 +3,57 @@
...
+    output = length * 2
+    pattern = /\h{#{output}}/
+  elsif format == "base64"
+    output = ((4 * length / 3) + 3) & ~3
+    pattern = /[a-zA-Z0-9+\/=]{#{output}}/
+  end
+  return pattern
+end
+
+def create_test_records(type:"all", number:2, session:"default")
```
[the remark this answers is not in the document]
Comment (dheles) [reply test/cli/test_helper.rb]: would you consider adding rubocop (and your preferred version) as a development dependency to the project's gemspec?

[the remark this answers is not in the document]
Comment (woodruffw) [reply test/cli/test_helper.rb]: Yeah. I'll add it now.

File: test/cli/.rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# this necessitates some complex methods
+Metrics/AbcSize:
+  Enabled: false
+
+# see above
+# this necessitates some complex methods
+Metrics/CyclomaticComplexity:
+  Enabled: false
+
+Style/RegexpLiteral:
```
Comment (woodruffw) [inline test/cli/.rubocop.yml]: Don't disable this check -- if you need inner slashes in a regexp, the `%r{}` literal syntax will be useful.

**Comment by dheles** [comment]
Are we just going to disregard the failed CI checks? I had been waiting for a resolution before proceeding.

File: test/cli/.rubocop.yml (hunk 2 above)
Comment (dheles) [reply test/cli/.rubocop.yml]: My thinking here is to silence complaints about existing code like this: 

```
  def test_session
    ...
    error_pattern_no_record = /No such record\(s\)/
    error_pattern_no_session = /Unknown session/
    ...
```
...where we are favoring the consistency of a single syntax for regexes at the cost of backslashes. I think what we have done reads better than mixing `/pattern/` and `%r{pattern}` throughout the code and definitely don't want to go back and change all the patterns that happen to contain a backslash. If we are happy with the code the way it is, then it makes sense for our rubocop rules to reflect that, doesn't it?

**Comment by woodruffw** [comment]
> Are we just going to disregard the failed CI checks?

Yeah. So long as it passes locally (and I can reproduce locally), we're fine. The CI is going to continue to fail until I come up with a way to securely share environment keys (which I'm working on).

File: test/cli/.rubocop.yml (hunk 2 above)
Comment (woodruffw) [reply test/cli/.rubocop.yml]: Yeah, that's reasonable. You can skip this, then.

File: test/cli/test_dump_fields.rb [hunk 3]
```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+require_relative "test_helper"
```
Comment (woodruffw) [inline test/cli/test_dump_fields.rb]: Change this to `require "helpers"` and remove the `require "aruba"` here and below, and this PR should be good to go.

**Comment by woodruffw** [comment]
Thanks @dheles!

**Pull request merged**